content/news/release-20150614.md

```diff
@@ -0,0 +1,55 @@
+We are proud to announce the June 2015 release of OpenRA!
+
+As promised, it was our goal to reduce the length of this development cycle, and we succeeded! With about 22500 changes lines in just over 300 commits by a total of 25 developers since last release, the list of new features is naturally a bit shorter than last time around, but it contains some good ones!
+
+The standout feature of this release is of course the long-awaited in-game map editor. For the first time ever, we are offering a map editing ability for all our mods, on all supported operating systems. You can access the map editor from the "Extras" sub-menu of the main menu.
+
+The new map editor offers the following benefits and features:
+
+* Fully cross-platform, just like the game itself, while the legacy editor did not work on OS X.
+* Truely WYSIWYG. It uses the game renderer, so there are no more rendering glitches, missing building roofs or turrets.
+* Support for all mods. The legacy editor did not support Dune 2000 properly, and Tiberian Sun not at all.
+
+Compared to the legacy editor, there is not yet full feature parity: the in-game editor is missing cut-and-paste operations, a display of the sum of resources on the map, and also does not show the coordinates of cells. These short-comings will be rectified in future releases. Meanwhile, they are still available in the legacy editor, which is still being shipped with this release. It will, however, be removed in the not too distant future.
+
+Complementing the new editor is an improved map chooser dialog, which now separates official and custom maps. It also allows deleting one or all of your custom maps. As an added goodie, the map chooser is now accessible to everybody from the lobby, not just the game host. Actually changing maps remains the prerogative of the host, of course.
```

Comment:
The structure of the last sentence feels a bit off.  How about changing the last two to "We also took this chance to add the much-requested ability for all players to view the map chooser from the game lobby (but changing the map still remains the prerogative of the host)."